examples/provider/provider.tf

```diff
@@ -1,8 +1,8 @@
 terraform {
   required_providers {
     nobl9 = {
-      source  = "nobl9.com/nobl9/nobl9"
-      version = "0.1.4"
+      source  = "nobl9/nobl9"
```

Comment:
source should contain  nobl9.com domain

Reply:
That's true for local builds, isn't it? In case someone wants to use a pre-built, released plugin provider (which I suppose could be more desirable for some people), I think the example shouldn't specify the hostname. In such case, terraform defaults to `registry.terraform.io`.

Reply:
`source  = "nobl9.com/nobl9/nobl9"` is for local purposes
It's a subpath for the plugin on our local machines ```~/.terraform.d/plugins/nobl9.com/nobl9/nobl9/...`

Reply:
Right, so it depends on whether in the "core example" for this repo we should point to:
- provider plugin built locally,
- pre-built provider plugin, released and present in registry.

I'd be more keen on the second option, but that's up to a debate. I'll just add that the example in `README.md` also uses the `source  = "nobl9/nobl9"`.